Snowflake.Data/Core/SFSession.cs

```diff
@@ -411,6 +411,8 @@ internal void startHeartBeatForThisSession()
             if (!this.isHeartBeatEnabled)
             {
                 HeartBeatBackground heartBeatBg = HeartBeatBackground.Instance;
+                if (this.masterValidityInSeconds == 0)
+                    this.masterValidityInSeconds = 14400;

```

Comment:
Could you please put comment here? why we change 0 to 14400 and what is 14400? Please use constant name instead of value.